recipe/recipe.yaml

```
@@ -1,14 +1,14 @@
 context:
   name: "awkward-cpp"
-  version: "46"
+  version: "47"
 
 package:
   name: ${{ name|lower }}
   version: ${{ version }}
 
 source:
   url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name|replace('-', '_') }}-${{ version }}.tar.gz
-  sha256: b82becb65a33280c6c27336ee5c037c5b7e394be52565aa31e087ca54adce59e
+  sha256: 676cf4976810edab32187edf5a8a716af95047b9038c96d27d3be44f1331950f
 
 build:
   number: 1
```

```suggestion
  number: 0
```

Bump this back to `0`

---

It works here: https://github.com/conda-forge/uproot-feedstock/pull/171/files though that was not from conda-forge-admin, it was the autotic bot (different title style), so maybe something is different there (other than time).